boards/nano-33-iot/definition.json

```diff
@@ -9,6 +9,18 @@
     "installMethod":"library",
     "components":{
        "digitalPins":[
+          {
+             "name":"TX",
```

Comment:
@tyeth  is there any `D` prefixed name for this, or is it just TX within the board definition?

Reply:
Looks like not:
https://github.com/espressif/arduino-esp32/blob/master/variants/nano32/pins_arduino.h
https://docs.arduino.cc/static/d77fe4114922e80804b6ebfd2e586316/ABX00027-pinout.png

Reply:
Looks like not, this is the pinout:
https://docs.arduino.cc/static/d77fe4114922e80804b6ebfd2e586316/ABX00027-pinout.png
This is the definition:
https://github.com/arduino/ArduinoCore-samd/blob/master/variants/nano_33_iot/variant.cpp#L35-L36

Technically we could support the second naming system of PBxx/PAxx for all pins, but recommend against that.